contracts/src/v0.8/dev/vendor/arbitrum/IArbitrumDelayedInbox.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MIT
```

Comment:
This file looks out of place, where is this interface coming from?

If this is the new interface from the updated version of the `arb-bridge-eth` vendor package, then please update the vendor contracts for that package there and use the existing structure.

Reply:
Left a comment about this below.  I'm open to better approaches if you have any.

Reply:
I think it's fine to have this interface here instead of importing the whole `Inbox` contract.